lib/src/gantt_default_week_header.dart

```diff
@@ -1,5 +1,14 @@
 import 'package:flutter/material.dart';
 
+// TODO extend display with week number
+extension WeeksBetween on DateTime {
+  int get weekNumberOfTheYear => (DateTime.utc(year, month, day)
```

Comment:
1. I suggest that instead of an extension method which might confuse users, you can add a function inside `GanttChartDefaultWeekHeader` , e.g. : `int getWeekNumberOfTheYear(DateTime src)`
2. please provide tests for this new method to check on edge cases like
  2.1. first week of the year
  2.2. middle week
  2.3. last week of the year

Reply:
well spotted, 1st week of the year required evol